src/frontend/src/components/DesignReviewStageGateModal.tsx

```diff
@@ -0,0 +1,43 @@
+import { useHistory } from 'react-router-dom';
+import { WbsNumber, ChangeRequestType } from 'shared';
+import { useCreateStageGateChangeRequest } from '../hooks/change-requests.hooks';
+import { useToast } from '../hooks/toasts.hooks';
+import StageGateWorkPackageModal from '../pages/WorkPackageDetailPage/StageGateWorkPackageModalContainer/StageGateWorkPackageModal';
+import { routes } from '../utils/routes';
+import { useCurrentUser } from '../hooks/users.hooks';
+
+interface StageGateModalProps {
+  wbsNum: WbsNumber;
+  modalShow: boolean;
+  handleClose: () => void;
+}
+
+interface FormInput {
```

Comment:
you should see if you can reuse `src\frontend\src\pages\WorkPackageDetailPage\StageGateWorkPackageModalContainer\StageGateWorkPackageModalContainer.tsx` here somehow. Maybe move that component to components and make it abstract enough so that it can be used in both places